q1_24.ts

```diff
@@ -0,0 +1,8 @@
+// 1.24 (*) Lotto: Draw N different random numbers from the set 1..M.
+import { integerFunction } from "./q1_22";
+import { randomElements } from "./q1_23";
+
+function randomNumbers(int1: number, int2: number, num: number) {
```

Comment:
If I gave input as (1, 5) and draw 4 random elements then will this function will return duplicate elements?